pom.xml

```diff
@@ -903,7 +909,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
                 <version>${maven-chckstyle.version}</version>
-
+                <configuration><skip>true</skip></configuration>
```

Comment:
Should whip this out. I think this was us temporarily turning off checkstyle.

Reply:
ah yes good spot! I've removed and resolved the checkstyle warning